packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -42,14 +47,30 @@ const _useOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedP
     remoteParticipants = [],
     localParticipant,
     dominantSpeakers = [],
-    maxRemoteVideoStreams,
+    maxRemoteVideoStreams = DEFAULT_MAX_VIDEO_SREAMS,
     maxOverflowGalleryDominantSpeakers = DEFAULT_MAX_OVERFLOW_GALLERY_DOMINANT_SPEAKERS,
     isScreenShareActive = false,
     pinnedParticipantUserIds = [],
     /* @conditional-compile-remove(gallery-layouts) */
     layout
   } = props;
 
+  const calculateMaxRemoteVideoStreams = (): number => {
+    /* @conditional-compile-remove(large-gallery) */
+    if (layout === 'largeGallery') {
+      return MAX_VIDEO_STREAMS_FOR_LARGE_GALLERY;
+    }
+    /* @conditional-compile-remove(gallery-layouts) */
+    if (maxRemoteVideoStreams > MAX_GRID_PARTICIPANTS_NOT_LARGE_GALLERY) {
+      return MAX_GRID_PARTICIPANTS_NOT_LARGE_GALLERY;
+    } else {
+      return maxRemoteVideoStreams;
+    }
+    return maxRemoteVideoStreams;
+  };
+
+  const maxRemoteVideoStreamsToUse = calculateMaxRemoteVideoStreams();
```

Comment:
Don't we calculate this in LargeGalleryLayout?

Reply:
this is to cap the number of tiles for when OVC eventually supports more than 9